CI-CD/AzureDevOps/azure-pipelines_dotnet_sbom.yml

```diff
@@ -53,5 +53,5 @@ steps:
     WS_APIKEY: $(APIKEY)
     WS_USERKEY: $(USERKEY)
 
-- publish: $(System.DefaultWorkingDirectory)/whitesource
-  artifact: Whitesource
\ No newline at end of file
+- publish: $(System.DefaultWorkingDirectory)/Mend
```

Comment:
Same as above, hard-coded or arbitrary name in this context?

Reply:
@lukebrogan-mend this should remain /whitesource as it is the folder for the unified agent scan...which I believe is still whitesource